miappe_v2.obo

```diff
@@ -0,0 +1,1257 @@
+comment: "Example: CC BY-SA 4.0, Unreported. Format: Unique identifier. Cardinality: 0-1"
+is_a: MIAPPE:0174 ! Investigation information
+
+[Term]
+id: MIAPPE:0007
+name: MIAPPE version
+def: "The version of MIAPPE used." []
+comment: "Example: 1.1. Format: Version number. Cardinality: 1"
+is_a: MIAPPE:0174 ! Investigation information
+
+[Term]
+id: MIAPPE:0008
+name: Associated publication
+def: "An identifier for a literature publication where the investigation is described. Use of DOIs is recommended." []
+comment: "Example: doi:10.1371/journal.pone.0071377. Format: DOI. Cardinality: 0+"
+is_a: MIAPPE:0174 ! Investigation information
+
+[Term]
+id: MIAPPE:0009
+name: obsolete_Study
+def: "A study (or experiment) comprises a series of assays (or measurements) of one or more types, undertaken to answer a particular biological question." []
+comment: "Cardinality: 1+ per investigation"
+is_a: ObsoleteClass ! Obsolete Class
+is_obsolete: true
+
+[Term]
+id: MIAPPE:0010
+name: Study unique ID
+def: "Unique identifier comprising the name or identifier for the institution/database hosting the submission of the study data, and the identifier of the study in that institution." []
+comment: "Example: EBI:12345678 http://phenome-fppn.fr/maugio/2013/t2351. Format: Unique identifier. Cardinality: 0-1"
+is_a: MIAPPE:0179 ! Study information
+
+[Term]
+id: MIAPPE:0011
+name: Study title
+def: "Human-readable text summarising the study." []
+comment: "Example: 2002 evaluation of flowering time for a panel of 375 maize lines at the experimental station of Maugio (France). Format: Free text (short). Cardinality: 1"
+is_a: MIAPPE:0179 ! Study information
+
+[Term]
+id: MIAPPE:0012
+name: Study description
+def: "Human-readable text describing the study." []
+comment: "Example: 2002 evaluation of male and female flowering time for a panel of 375 maize lines representing the worldwide genetic diversity at the experimental station of Maugio, France. Format: Free text. Cardinality: 0-1"
+is_a: MIAPPE:0179 ! Study information
+
+[Term]
+id: MIAPPE:0013
+name: Start date of study
+def: "Date and, if relevant, time when the experiment started." []
+comment: "Example: 2002-04-04 2006-09-27T10:23:21+00:00. Format: Date/Time (ISO 8601, optional time zone). Cardinality: 1"
+is_a: MIAPPE:0179 ! Study information
+
+[Term]
+id: MIAPPE:0014
+name: End date of study
+def: "Date and, if relevant, time when the experiment ended." []
+comment: "Example: 2002-11-27. Format: Date/Time (ISO 8601, optional time zone). Cardinality: 0-1"
+is_a: MIAPPE:0179 ! Study information
+
+[Term]
+id: MIAPPE:0015
+name: Contact institution
+def: "Name and address of the institution responsible for the study." []
+comment: "Example: UMR de Génétique Végétale, INRA – Université Paris-Sud – CNRS, Gif-sur-Yvette, France. Format: Free text (short). Cardinality: 1"
```

Comment:
To avoid confusion, should we use a consistent delimiter between different options in examples, or limit each example to a single instance? The "-" in the current format could be misunderstood as either separating different inputs or connecting multiple institutions in a single input.

Reply:
Technically the cardinality for this field is one, so there should only be one entry for contact institution, not multiple

Reply:
In the MIAPPE file they often gave multiple examples so they are all transferred. I would suggest to separate them by ";"? We can also decide to only keep one, but I think e.g. for the examples above (date and license) it can be helpful to see different possibilities.

Reply:
I think it would be clearer, if the examples includes also the name for example: 
comment: "Example\: 'Contact institution\: UMR de Génétique Végétale, INRA.'; 'Contact institution\: Université Paris-Sud, INRA.'; 'Contact institution\: Gif-sur-Yvette, France.'; Format\: Free text (short). Cardinality\: 1"
Just found out that maybe the colon ":" needs to be escaped according to the obo format.
https://owlcollab.github.io/oboformat/doc/GO.format.obo-1_2.html#S.1.2

Reply:
Maybe a compromise would be to not include the name but do use the single quotes as you did and change it to exampleS, so it's clear that it is multiple:
comment: "Examples: 'UMR de Génétique Végétale, INRA.'; 'Université Paris-Sud, INRA.'; 'Gif-sur-Yvette, France.'; Format: Free text (short). Cardinality: 1"
Protege did not give any errors, so I think using the colon is ok(?)

Reply:
Yes good idea :D